curated_transformers/tokenization/bert_tokenizer.py

```diff
@@ -300,6 +301,13 @@ def from_files(
             special_pieces=special_pieces,
         )
 
+    @property
+    def eos_piece(self) -> Optional[str]:
+        return self._eos_piece
+
+    def piece_to_id(self, piece: str) -> Optional[int]:
+        return self.processor.piece_to_id(piece)
+
```

Comment:
Can't we move this to `WordPieceTokenizer`?

Reply:
Moved.